apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -297,6 +308,39 @@ class LivechatClass {
 		});
 		return true;
 	}
+
+	sendRequest(postData: LivechatSendRequestData, attempts = 10) {
+		if (!attempts) {
+			return;
+		}
+		const timeout = settings.get<number>('Livechat_http_timeout');
+		const secretToken = settings.get<string>('Livechat_secret_token');
+		const headers = { 'X-RocketChat-Livechat-Token': secretToken };
+		const options: HTTP.HTTPRequest = {
+			data: postData,
+			...(secretToken !== '' && secretToken !== undefined && { headers }),
+			timeout,
+		};
+		try {
+			const result = HTTP.post(settings.get('Livechat_webhookUrl'), options);
+			if (result.statusCode === 200) {
+				metrics.totalLivechatWebhooksSuccess.inc();
+			} else {
+				metrics.totalLivechatWebhooksFailures.inc();
+			}
+			return result;
+		} catch (err) {
+			Livechat.webhookLogger.error({ msg: `Response error on ${11 - attempts} try ->`, err });
+			// try 10 times after 20 seconds each
+			attempts - 1 && Livechat.webhookLogger.warn(`Will try again in ${(timeout / 1000) * 4} seconds ...`);
+			setTimeout(
+				Meteor.bindEnvironment(function () {
```

Comment:
I'm gonna ask, do we _actually_ need this meteor.bindenv? don't see any meteor thing going on in here apart from http 👀

Reply:
We can try removing it and doing the request flow

Reply:
it seems it was not needed at all